src/App.js

```diff
@@ -1,14 +1,20 @@
+import "./App.css";
 import { Route, Routes } from "react-router-dom";
-import { HomeScreen } from "./screens";
+import { HomeScreen, ProductsScreen } from "./screens";
 import { Footer, Navbar } from "./components";
 const App = () => {
   return (
     <>
-      <Navbar />
-      <Routes>
-        <Route path="/" exact element={<HomeScreen />} />
-      </Routes>
-      <Footer />
+      <div className="body-wrapper">
+        <Navbar />
+        <Routes>
+          <Route path="/" exact element={<HomeScreen />} />
+        </Routes>
+        <Routes>
+          <Route path="/products" exact element={<ProductsScreen />} />
+        </Routes>
```

Comment:
You can refactor this to-
<Routes>
          <Route path="/" exact element={<HomeScreen />} />
          <Route path="/products" exact element={<ProductsScreen />} />
</Routes>

Reply:
ok cool.